src/app/+item-page/simple/item-types/shared/item-relationships-utils.ts

```diff
@@ -43,26 +51,32 @@ export const compareArraysUsingIds = <T extends { id: string }>() =>
 /**
  * Fetch the relationships which match the type label given
  * @param {string} label      Type label
+ * @param side                Which side to filter the relationships on: left/right (takes both sides when not provided)
  * @returns {(source: Observable<[Relationship[] , RelationshipType[]]>) => Observable<Relationship[]>}
  */
-export const filterRelationsByTypeLabel = (label: string) =>
+export const filterRelationsByTypeLabel = (label: string, side?: RelationshipSide) =>
   (source: Observable<[Relationship[], RelationshipType[]]>): Observable<Relationship[]> =>
     source.pipe(
       map(([relsCurrentPage, relTypesCurrentPage]) =>
         relsCurrentPage.filter((rel: Relationship, idx: number) =>
-          hasValue(relTypesCurrentPage[idx]) && (relTypesCurrentPage[idx].leftLabel === label ||
-          relTypesCurrentPage[idx].rightLabel === label)
+          hasValue(relTypesCurrentPage[idx]) && (
+            (hasNoValue(side) && (relTypesCurrentPage[idx].leftLabel === label ||
+              relTypesCurrentPage[idx].rightLabel === label)) ||
```

Comment:
It's best not to work with a side parameter, but rather to check the label vs the item ID:
* If the current item occurs on the left, the label should match the left one OR
* if the current item occurs on the right, the label should match the right one

It would also be useful to add some comments explaining this behavior

Reply:
As requested, I've removed the RelationshipSide enum and replaced it with the item's ID. Using the item's ID, it now filters out irrelevant relationships where the item is on the same side as the label requested.
I've also updated the PR's description to match this change.